UI.tsv

```diff
@@ -568,62 +568,62 @@ UI_20150317_000567	{@st41}Damage Top 100{/}
 UI_20150317_000568	{@st41}Money Money Top 100{/}
 UI_20150317_000569	{@st41}Level Up Top 100{/}
 UI_20150317_000570	{@st41b}Menu{/}
-UI_20150317_000571	{@st43}{s22}Enhance Item{/}
+UI_20150317_000571	{@st43}{s22}Refine Item{/}
```

Comment:
There is nothing wrong with the previous. 'Enhance' is more commonly used and known.